spec/at-rules/for.md

```diff
@@ -22,38 +22,38 @@
 
 To execute a `@for` rule `rule`:
 
-* Let `from` be the result of evaluating the expression in `FromDeclaration`
-
-* Assert that `from` is a number
+* Let `from` be the result of evaluating the expression in `FromDeclaration`.
 
 * If `rule` has a `ToDeclaration`:
 
-  * Let `to` be the result of evaluating the expression in `ToDeclaration`
+  * Let `to` be the result of evaluating the expression in `ToDeclaration`.
 
-  * Let `exclusive` be `true`
+  * Let `exclusive` be `true`.
   
 * Otherwise:
 
-  * Let `to` be the result of evaluating the expression in `ThroughDeclaration`
+  * Let `to` be the result of evaluating the expression in `ThroughDeclaration`.
 
-  * Let `exclusive` be `false`
+  * Let `exclusive` be `false`.
   
-* Assert that `to` is a number
```

Comment:
having separate steps for number and integers was done on purpose due to the unit conversion that will happen between them (in the fast-track proposal specifying the handling of units).

Reply:
All right, feel free to add it back.